supporting/Dockerfile

```diff
@@ -3,7 +3,7 @@ USER root
 RUN wget -q -O /tmp/mrutter.asc "https://keyserver.ubuntu.com/pks/lookup?op=get&search=0xe298a3a825c0d65dfd57cbb651716619e084dab9" && \
     sudo apt-key add /tmp/mrutter.asc && \
     sudo apt-get update && \
-    sudo apt-get install texmaker -y
+    sudo DEBIAN_FRONTEND=noninteractive apt-get install texmaker -y
```

Comment:
The '-y' should be skipping the confirmation prompt already. But this change looks fine to me otherwise